evm/src/cpu/kernel/constants/mod.rs

```diff
@@ -86,12 +87,23 @@ pub(crate) fn evm_constants() -> HashMap<String, U256> {
     c
 }
 
-const MISC_CONSTANTS: [(&str, [u8; 32]); 1] = [
+const MISC_CONSTANTS: [(&str, [u8; 32]); 3] = [
     // Base for limbs used in bignum arithmetic.
     (
         "BIGNUM_LIMB_BASE",
         hex!("0000000000000000000000000000000100000000000000000000000000000000"),
     ),
+    // Position in SEGMENT_RLP_RAW where the empty node encoding is stored. It is
+    // equal to usize::MAX so that all rlp pointers all much smaller than that.
```

Comment:
```suggestion
    // equal to usize::MAX so that all rlp pointers are much smaller than that.
```

Reply:
it also still mentions `usize` while `u64` may be better (not platform specific) even if you're using `usize` when comparing with `0xFFFFFFFF` below? non-blocking though